frontend/src/main/components/Commons/CommonsPlay.js

```diff
@@ -1,13 +1,25 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
+import greetingsList from "../../../assets/PlayGreetings.json"
 
 export default function CommonsPlay({ currentUser }) {
-  // Stryker disable next-line OptionalChaining : too many cases to test for too little value
+  // Stryker disable  all 
   const firstName = currentUser?.root ? currentUser?.root?.user?.givenName : "";
+
+  const [welcomeText, setWelcomeText]= useState("Welcome farmer");
+
+  useEffect(() => {
```

Comment:
Don't use an effect here. Calculate the random message in useState's initializer callback. Otherwise it will trigger an unnecessary render cycle and you get a flash of stale state.

Reply:
Thanks for the feedback! Fixed in [af19144](https://github.com/ucsb-cs156-m23/proj-happycows-m23-10am-4/pull/36/commits/af19144445c862664e6c41afd3e6e8083e77538c)